lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskCommentServiceImpl.java

```diff
@@ -69,13 +69,13 @@ TaskComment updateTaskComment(TaskComment taskCommentToUpdate)
 
       taskService.getTask(taskCommentImplToUpdate.getTaskId());
 
-      if (taskCommentToUpdate.getCreator().equals(userId)
+      TaskComment originalTaskComment = getTaskComment(taskCommentImplToUpdate.getId());
```

Comment:
We already do a getTask() in getTaskComment() (I guess in order to check whether the user is authorized to see the task?) In this case the getTask() in line 70 is unnecessary, isn't it?

Reply:
That's true. Removed the additional getTask call from line 70